src/test/java/org/opensearch/ad/transport/ADResultBulkTransportActionTests.java

```diff
@@ -8,40 +8,9 @@
  * Modifications Copyright OpenSearch Contributors. See
  * GitHub history for details.
  */
-
+/* @anomaly.detection Commented until we have extension support for hashring : https://github.com/opensearch-project/opensearch-sdk-java/issues/200 (needs AnomalyResultTests class)
```

Comment:
Why are we commenting the whole test file for hashring? Can't we just pass null for hashring or use extensioNode instead of making request to all the nodes?

Reply:
I can handle these changes in the multi-entity results PR. As part of that PR I have modified the `ADResultBulkTransportAction` and removed the indexing pressure object from that class so some of these tests will have to be rewritten

Reply:
That works